.env.example

```diff
@@ -0,0 +1,4 @@
+#Environment file for docker-compose
+
+PATH_OMS_GLOBAL=oms-global/docker/
+PATH_OMS_CORE=oms-core/docker/
```

Comment:
I suggest looking into this combined with the general config/environment rework you were thinking of.